### ISSUE TITLE
Is there any shareable spec06/spec17 on unmatched

### ISSUE THREAD

**Issue by fanghuaqi** [mollybuild/RISCV-Measurement#3]
Hi there, I see you have already post some steps to run spec2006 and spec2017, and is there any shareable results for your evaluation on  unmatched?

**Comment by mollybuild** [comment]
> 

Yes, I update in PLCT lab Weekly report which publishes monthly:

https://github.com/plctlab/PLCT-Weekly/blob/master/2022/2022-03-01.md#riscv%E5%B9%B3%E5%8F%B0%E6%B5%8B%E8%AF%95%E6%B5%8B%E8%AF%84%E5%B7%A5%E4%BD%9C
